pkg/util/goroutinemap/common/errors.go

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
I don't think you need this file. I would just move all the exponential errors into the exponential file and then just copy the already exists functionality.